packages/terra-consumer-site/src/site.scss

```diff
@@ -0,0 +1,17 @@
+// Styles specific to development only
+// stylelint-disable
+
+#root {
+  padding: 20px;
+}
+
+.markdown-body {
```

Comment:
You can remove these styles. They should be moved to the markdown component now.